### DISCUSSION TITLE
Weblate translations instead of Crowdin

### DISCUSSION THREAD

**Discussion by comradekingu** [NightscoutFoundation/xDrip#3420] [Ideas]
Would be a nice gesture to translators to not expose them to the invasive terms and services of Crowdin,
and rather use a tool that allows spanning multiple languages and strings quickly and at a glance.
Lots of similar projects on Hosted, like for example https://hosted.weblate.org/projects/ubpm/ :)

**Comment by maboroshin** [comment]
Crowdin can work in one window. Search and listing and other languages are always in the same window.

Weblate separates the listing function from the other functions. It's not possible to refer to other languages from list mode. Multiple strings and multiple languages are always separate windows. I don't think these will ever be merged into one window.

* See UI details: https://github.com/maboroshin/translation/wiki/UI
* Here is a summary of objective data. https://github.com/maboroshin/translation/issues/1

Crowdin:
![image](https://github.com/user-attachments/assets/00daaaa4-0d2b-4d1e-bf34-3eb129b71ac4)

----

Listing on Weblate:
![image](https://github.com/user-attachments/assets/b7277a78-4b32-442c-9c9d-821b83737bc4)

----

Multiple languages  on Weblate:
![image](https://github.com/user-attachments/assets/d08df22e-49eb-4db9-9858-c75b1e523b67)

**Reply by comradekingu** [reply]
You haven't set up any more languages in the Zen editor of Weblate and post cropped screenshots to avoid giving credence to the still factual statement that one platform paginates strings, whereas the other allows browsing all strings at once.
Then you link to your own easily debunked "research" without giving fair warning, which is both wrong and already debunked in the thread.

"fantasy" and "perceives reality in reverse" isn't quite dealing with the the issues of privacy or what is effectively sub-licensing of commissioned work, but if you can't produce sound arguments there is always that level of irony to deal in.

Weblate is without question the de-facto choice for libre software translation.

**Reply by maboroshin** [reply]
Hmmm, I don't see the point of showing other languages in the listing. That would make it very difficult to use. Because, there is the "Other Languages" tab for convenience.

To use functions other than translation, such as other languages, history reference, etc. The page must be loaded with "forward" and "back" for each string on Weblate. Weblate doesn't integrate listings. Otherwise, multiple windows are always necessary. In this case, strings translated in one window are not synchronized to other windows.

Please rerrer to actual data.

**Reply by comradekingu** [reply]
So because you don't understand the utility, in something that you haven't used, you are also an expert on its use.
Real convenient to not have it all in one place.
Strings not all in the same place, and other languages not in that place.

Your logic is good, you just don't apply it to what you use it for.

How are other languages not translation?
If being able to fix all strings in one place, it follows being able to fix other languages from the same place is good.

You are saying pagination isn't good, I agree. Therefore Crowdin and Transifex don't cut it.

>The page must be loaded with "forward" and "back" for each string on Weblate. Weblate doesn't integrate listings.

This is the entire point of the Matrix and Zen views.

It isn't just one editor that sucks, and you haven't answered to the problem yet.
Nor have you done anything to refute an even more poignant and central point.
Crowdin and Transifex connect to all sorts of domains, send data in all directions, and makes all sorts of provisions for itself to monetize this data in all sorts of ways.

You are making this argument for something that is only available as a closed source software-as-as-service system, in a full and strong copylefted libre software project.

Crowdin won't even let anyone see the strings without logging in. There is a reason the libre software world has rejected it.

**Reply by maboroshin** [reply]
I use all the services. Is a huge number of clicks "real convenient"? There is no reason for your fantasy.

As in the image above, on Crowdin, selecting a single string displays all of its history, comments, other languages, etc. It is an excellent UI for collaboration.

To move 10 strings ahead, one click from the left side bar on Crowdin.

Open source strings can also be read from that repository.

----

But in Weblate, you have to press the Comments tab, the Other Languages tab, and the History tab, respectively. Moreover, this window must be opened separately from the list window. This is probably why other users at Weblate often don't read the comments.

Weblate's "forward" and "back" load one page at a time, making it very difficult to use. Press Forward 10 times to move 10 strings. This UI is unusable because that alone takes 10-20 seconds.

----

New topics by comradekingu have been separated below: https://github.com/NightscoutFoundation/xDrip/discussions/3420#discussioncomment-10508957

**Reply by maboroshin** [reply]
(Let's separate the topics) comradekingu replied with this below: https://github.com/NightscoutFoundation/xDrip/discussions/3420#discussioncomment-10509327

> There are fewer clicks because you can translate a whole component without the pagination.
> 
> Earlier you wrote
> 
>     Multiple strings and multiple languages are always separate windows.
> 
> When Weblate is the only platform that allows this…
> What are you missing?
> 
> That is before you didn't know how to configure it, didn't understand what that would do, and concluded it was useless.
> Now you are just right back to the belief that fewer clicks are to be had from pagination.
>
> This is simply not true from a simple listing of all strings in a component available on one page, (which can be > had with the Zen and Matrix editors in Weblate),
> but it is even more true if you factor in looking at other languages.
>
> Weblate is vastly more popular for libre software than Crowdin, with or without my help.
>
> Crowdin has a lot of the same problem Transifex does. When you use part of the width of the screen badly, it both line-shifts the source string regardless of where actual lineshifts are, but with varying verbosity of translations, it compounds into line-shifts that are even different from the source string.
> Spanning between these is very taxing, and on a long list of things that makes translation harder than it has to be with Crowdin and Transifex.

**Reply by maboroshin** [reply]
I have already explained. In Weblate, history, comments, other languages, each requires a click, making it difficult to use. This probably makes it difficult to collaborate. I also provide images.

Crowdin is all in one window. Weblate has too much white space. So lead more clicking and scrolling. Weblate would be difficult to use for the purpose of entering or collaborating on translations.

----

> Weblate is vastly more popular for libre software than Crowdin

There is no data of any kind. I previously searched for open projects on Crowdin: https://crowdin.com/projects  Crowdin had more than 16,000. In contrast, on Hosted Weblate, it's over 1,100: https://hosted.weblate.org/projects/?page=12&limit=100 Where does the data come from?

----

> Crowdin has a lot of the...

Hmm, Go back to the beginning.

> spanning multiple languages and strings quickly and at a glance.

Go back to the beginning. As I have shown, it's difficult to use. Please explain your claim with pictures.

**Reply by comradekingu** [reply]
These are some of the things you just don't understand:
Either what libre software is, how it compares to closed source software.
That Weblate is both a codebase that is freely (GPLv3+) available for any use, and a service.
That it is not my job to disabuse you of your position also in picture form.

"Spanning multiple languages and strings quickly and at a glance" ultimately means being able to see all languages and strings of a component at once. It isn't a hard concept to grasp. Neither is what not having that entails.

I even detailed the sequence of events that lead to your lets at best call that confusion.
Do you think I owe you something? I don't.

In my generosity, I bright up another salient point that you also could have understood and instead you go with
>Hmm, Go back to the beginning.

Let's apply your own logic, because it is still that you are operating within the irony of its failed application.

I am in a libre software project I want to contribute to without agreeing to invasive terms and conditions, and you
simply fail to reason with that point.

The page you linked to says
>We value your privacy
We use cookies to enhance your browsing experience, serve personalized ads or content, and analyze our traffic.

**In what universe does a simple table listing need 67 (I counted them) cookies?
You want to pretend it is even close to the position of a rational actor that a company that does this has values that align with xDrip, or any user you could even imagine for the purposes of just listing information?

You are in a libre software project refusing to read the terms and services this discussion is about,
talking about functionality you evidently haven't used.

**Comment by maboroshin** [comment]
comradekingu replied above: (New topic)

> Crowdin and Transifex connect to all sorts of domains, send data in all directions, and makes all sorts of provisions for itself to monetize this data in all sorts of ways.

> You are making this argument for something that is only available as a closed source software-as-as-service system, in a full and strong copylefted libre software project.

> Crowdin won't even let anyone see the strings without logging in. There is a reason the libre software world has rejected it.

----

> There is a reason the libre software world has rejected it.

You are the individual named "comradekingu". You are not "libre software world". And, most of the spam against your libre software that suggests using Weblate has been rejected.

----

Weblate is not transmitting data? weblate is not disclosing details of how it handles information. We don't know because Weblate doesn't have the transparency of other services.

Weblate doesn't use the data for monetization, does it? weblate is one of the profitable companies. Where does your fantasy come from?

(Note: I moved  text about usability to the thread above)

**Reply by comradekingu** [reply]
There are fewer clicks because you can translate a whole component without the pagination.

Earlier you wrote

>Multiple strings and multiple languages are always separate windows. 

When Weblate is the only platform that allows this…
What are you missing?

That is before you didn't know how to configure it, didn't understand what that would do, and concluded it was useless.
Now you are just right back to the belief that fewer clicks are to be had from pagination.

This is simply not true from a simple listing of all strings in a component available on one page, (which can be had with the Zen and Matrix editors in Weblate),
but it is even more true if you factor in looking at other languages.

Weblate is _vastly_ more popular for libre software than Crowdin, with or without my help.

Crowdin has a lot of the same problem Transifex does. When you use part of the width of the screen badly, it both line-shifts the source string regardless of where actual lineshifts are, but with varying verbosity of translations, it compounds into line-shifts that are even different from the source string.
Spanning between these is very taxing, and on a long list of things that makes translation harder than it has to be with Crowdin and Transifex.

I know exactly what Weblate is doing because the entire codebase is libre software.
The terms and services are short and written in plain English https://hosted.weblate.org/legal/
If you think Weblate does anything that isn't in there we all want to know.
And you have exactly this problem, it is an baseless accusation, because the provisions aren't in there.
That means either using work that is copyrightable outside of licensing, or using personal data in ways that are not described.
Both are illegal in the jurisdiction Weblate operates in.

Not only is the Weblate service _vastly_ superior in terms of privacy, it isn't a secret either.
Same goes for other instances, like LibreOffice and such.
Go to https://hosted.weblate.org/legal/ and notice it only connects to the first party domain.

You seem to not notice Weblate is both a software company, and a software service.
If you want to run the software yourself without the service, you have total control.
It can't logically get better than that.

The logical opposite is a closed source software with endless terms and services, that connects to everyone and their dog.
That is what Crowdin is. That is what Transifex is.

>We don't know because Weblate doesn't have the transparency of other services.

Please detail what you actually mean here. You couldn't be more wrong if you tried.
What isn't transparent at Weblate/with Weblate, and how do you come to that from defending Crowdin of all things?

Again, I implore you to actually read the terms and services of each service, because it really is night and day.
If you don't understand what I write, there is no chance you will understand the legalese in the Crowdin and TX terms.
They are written that way because it isn't intended to be read by users, and it is not the friend of the user.
The contrast couldn't be more stark.
Length, complexity, scope, there is no dimension in which the Weblate terms and services isn't what the user would want.

**Reply by maboroshin** [reply]
Let's go back to the thread above about usability.

----

The lack of explicit consent for the handling of personal information, such as email address leaks at Weblate, has been reported many times. [Some were concerned about its illegality](https://github.com/WeblateOrg/weblate/issues/6508#issue-987139654).

* https://github.com/WeblateOrg/weblate/issues?q=is%3Aissue+GDPR

* https://community.crowdin.com/search?q=GDPR Nothing to report on Crowdin. 
* https://community.transifex.com/search?q=GDPR Nothing to report on Transifex.

----

Companies that comply with the GDPR describe their data handling practices in detail.

* Weblate only says that [cookies are to be used for content personalization](https://weblate.org/terms/).
* In contrast, [Transifex's cookie policy](https://www.transifex.com/legal/cookies-policy/) is very detailed. And importantly, you can choose not to use cookies at first. This is called the "opt-in". The opt-in is that it's GDPR compliant.
* [Crowdin's cookie policy](https://support.crowdin.com/cookies/)  is also detailed. There is a setting at the bottom of this page to turn off cookies. Crowdin also has opt-in.

**Comment by Navid200** [comment]
Please be kind.  Please focus on the subject of the discussion.  
It's perfectly fine to disagree.  There is nothing wrong with having a disagreement.  But, please don't make it personal.
Thanks so much for being kind.

**Reply by comradekingu** [reply]
I'll read personal descriptions drawn out of thin air all day. Don't worry about me :)
It does humor me greatly at times, and there is some schadenfreude in having that—bereft of actual productive responses,
but it is also saddening to see the distractions people arrive at in the pursuit of saving an assumed position.
It actually scares me to catch a glimpse of such a mind.
I don't want to acknowledge a world where people are like this, and it is in some way something I could face one day.
There is a lot of humility to be found in that the same could be true of me in the eyes of people more capable than myself.

**Comment by Navid200** [comment]
@maboroshin This discussion started by @comradekingu .
Everyone is welcome to start a conversation here.

Your first post in this discussion started by saying:
_comradekingu posts fantasy. And comradekingu perceives reality in reverse._ 
 
That is a statement about him not about the subject of this discussion.  
For the last time, I ask you to stop making personal attacks.
If you cannot do that, don't make any more posts here.  

I have no idea what this conversation is about.  Therefore, I neither agree nor disagree with what his first post was about.
But, I can see you made it personal.  You are not supposed to do that.  

You can disagree with someone without making it personal.  If the only way you can communicate is by making personal attacks, please stop posting.  I am so sorry.  But, this has to end.
It is clear you disagree with him.  You have made your point.  You have to allow him to defend his position.
But, you cannot make it personal.  It will  become ugly.  it will be come tainted.  Please end this.

**Reply by maboroshin** [reply]
Sorry. Let me erase that sentence. That is not important, so I erased it. Please don't hesitate to let me know.

**Reply by comradekingu** [reply]
When you are ponderous about being bored with being bothered the posts gets long, the post:

I have been introspecting myself into thinking I am wrong with the highest level of introspection I can do.
It is a minute thing really, but who could be this wrong about something where the logic could only be faintly perceptible.
Who am I to say there isn't an equal and inverse, but also aggrandising misunderstanding on my part.
Just don't ban the guy.
Yes, he fails to take you up on your wish to banish in himself, but there is a consideration for improvement at some point.
As a concept I believe in the idea of assuming guilt in some display of repentance, and then correcting behavior and not just prior posts when that should have been the case.

Do I have any hope this will happen? Well, maybe more of a discipline of necessity in this case.
I have a lot of faith in symbiotic people, and the software you make here struggles to find competition in scenarios one could even dream up.

I am very unabashed about what washes out as just personal motivation in projected diamentrality, but it tells a story.
It is one of a person whom in their own mind was thought to be correct, and then started that sequence of communicating with the outside world a giant leap of faith as to whether it would hold up.

You rob that world of seeing in someone how someone who would do that in a discussion as a psychology experiment, if the action is aggrevated, diminished and appearing serious enough, I think openness is most all we have.

To take that away would mean it is actually endemic to the individual beyond the recourse any libre software project for any reason one could muster up the required involvement to get going.

The layered level of unification means other people get to glaze directly into the dim decision process of someone who shows up as a not only repeating, but also an unrepenting. Shooe the unrepenting!

If I or others have to deal with who we don't want to be because we can't deal with that (a belief strengthened by both prior experience as all but the slimmest inability to rationally predict), the layer of communication could be called into question.
, but also then we send things we see in ourselves onwards.
Only in the things the most clever can't see in themselves should we ban ourselves from thinking of whom to consider others.
We shall know they are rare occations manifest.

It is easy to get saddened by sub-optimal behavior, and it is easy to fall into the as such mind-trap of thinking all are the same.
If one is to have "all the users", on an equal footing that means I will have all users I don't want others to have to deal with,
because I should let others have just one more dig at me before hitting that button.
Otherwise I wish the behavior I fail to single-handedly correct be at the mercy of others of similar kind, at the closest of its relative distance— both in owns own project, but also as an assimilate of cover for the ecosystem of individuals who share the thinking and passion for maintaining the operation of such an ecosystem at all levels thereby and as I am thick-skinned, but not when applying empathy for how you may not. It is a scary removal-process.
That is to say I don't know you — as I know myself, but I think you are a great guy for good reason.

I hope to show there are some people you come into contact with in the little world that is created in terms of ones own project,
but also its perception. Some people at least aspire to be a person of great stature, which I would say in your image—at that.

And only at that level do you even believe that the cognitively uncorrectable is so far beyond your own comprehension not only as to be just one layer beneath it.
And then you can breathe in all circumstances and never forget.

<details>  <spoiler\>A: Haha, I tried to catch a guy in another dimension.
                   Deplating, that is deep irony. >/spoiler :<<details>
                   
                   That is how we issue our appologies.
                   
                   Just because you think something is dumb, does not mean it could perceptibly not be dumber in you.
                   We can be all but the outmoust dumb in where to place the immovably dumb in having a limit.

So embarassment is all we are competing over here really, and that is pretty silly.
I on both hands have to battle with my deepest concepts of understanding as a concept and place them against the introspective level of how wrong this guy is to even understand. 

It is like being scared of not being able to do simple things.
I have to start believing in straight multidimensional AI for that to make sense.
And I know that doesn't make sense, but I have to believe what doesn't make sense to see how senseless it is.
All emotion is tied to perception, and control over it is tying those motorskills to behaviors.
Therefore thought comes from mastering motorskills with someone.
It is learnt behavior of joy.
Sudden movement causes shock.

I can't but help feel sorry for what would otherwise be mental capacity.
That is the meaning of that sorry-looking content above, post.

**Comment by Navid200** [comment]
@comradekingu Are you an xDrip translator?

I volunteered to be.  Then, I logged in and attempted to translate.  And I did a few words.   Then, I realized I was the wrong person.
I left my country of birth a long time ago before there were smart phones.  When I attempt to translate, I realize that I don't know what people speaking that language use for words like tap, or search or enter or ...

I volunteered because I wanted to improve xDrip for those speaking that particular language.  But, I realized I was not qualified.  So, the short story is that I never did a lot of translation in Crowdin.
But, I did not notice any invasive terms of service.  
What exactly are those?

Even if you get agreement from everyone here, it may not be very easy to switch from one utility to another.  So, it may help if you clarify what exactly the problem is so that the person who has to make the decision can weigh the pros and cons.

**Reply by comradekingu** [reply]
That makes more sense, because I would think the maintainer to be much younger and then different characteristics would apply. The maintainer is probably not reading this (for the better at this point), but those people really do understand very simple instructions.
It isn't that hard to switch technically.
I can find more details that are easy to understand, but that would be just for you at this point.

First copyright is established.

>7.1 Uploading Client Data to Platform.

>If the Client uploads **Client Data to the Platform, such Client Data and any processing of such Client Data must be in compliance with these Terms** and applicable law. By uploading Client Data to the Platform and/or using Crowdin Services, the Client represents that the Client has **all necessary legal rights**, title and interest **in and to the Client Data** whether posted and/or uploaded by the Client or with Client’s authorization or made available on or through the Crowdin Services by Supplier. By uploading Client Data to the Platform, **Client authorizes Supplier to process the Client Data**.

>For the purposes of Article 28 of Regulation (EU) 2016/679, these Terms constitute the data processing contract between the Client as the data controller and the Supplier as the data processor. The Client hereby instructs the Supplier to process the data as described in these Terms.

>(Then it is protected against malice and a lof of other things that could give the platform indemnification even if those possibly were to happen.)

This is what loses effective copyright:

>the Client and any of the Users associated with the Account do not create, transmit, display or make otherwise available any Client Data that violates the terms of these Terms

>9.2 Duration.

>The Supplier will process data on behalf of the Client until the termination of the Crowdin Services in accordance with these Terms. Upon termination, Crowdin will store the Client’s data for a period of six months, should the Client wish to reopen the Account to resume the use of the Crowdin Services or to export Client Data, unless instructed otherwise by the Client. 

Then what is agreed to

>Special Terms for Crowdin Enterprise Users. (For Crowdin Users Only) As part of your voluntary contribution to any Crowdin project, by agreeing to these terms, you are acknowledging and agreeing that your name, username and email address will become embedded and part of the project and organization, which may be available to organization admins. You understand the removal of this information would be impermissibly destructive to the project and the interests of all those who contribute, utilize, and benefit from it. Therefore, in consideration of your participation in any project, you understand that retaining your name, username and email address, as described above, does not require your consent and that the right of erasure, as spelled out in the GDPR Article 17 (1) b does not apply. The legal basis for our lawful processing of this personal data is Article 6 (1) f (“processing is necessary for the purposes of the legitimate interests pursued by the controller”).

It is there, it has your name on it, and they say they can process it.
It isn't the client data in terms of name and e-mail, because of course you should have the rights to redistribute that.

"As part of your voluntary contribution" could mean also the voluntary nature of giving crowdin access to something else that is in it.

28 is this

(28) The application of pseudonymisation to personal data can reduce the risks to the data subjects concerned and
help controllers and processors to meet their data-protection obligations. The explicit introduction of ‘pseudony
misation’ in this Regulation is not intended to preclude any other measures of data protection.

They call it pseudonomisation, but want to use it for the opposite.
Just to say it is processing, of client data, for legitimate interests.

So we find out what is legitimate about those.

" interests of all those … and benefit from it."

Client data is not personal data one could argue, but lets make extra sure:

https://support.crowdin.com/privacy-policy/ Client Data 	personal data, reports, addresses, **and other files**, folders or **documents in electronic form that a User of the Service stores within the Service**.

The processing is subject to storage.

They grant themselves the right to protect the copyright:

>3.3 Communications

**We** may **use** a Visitor’s or User’s email address or **other information – other than Client Data** – to contact that Visitor or User (i) for administrative purposes such as customer service, **to address intellectual property infringement**, right of privacy violations or defamation issues related to the Client Data or Personal Data posted on the Service or (ii) with updates on promotions and events, relating to products and **services offered by us and by third parties we work with**.


>**We** use **automatically collect**ed **information and other information collected on the Service through cookies and similar technologies** to: (i) personalize our Service, **such as remembering a User’s or Visitor’s information** so that the User or Visitor will not have to re-enter it during a visit or on subsequent visits; (ii) **provide customized advertisements, content, and information;**

I don't want customized adverts provided to me because I used a service.
Nowhere and at no time.
They provide "information".

So we just need to tie the two documents together then, and guard against all loss of any sort of revenue possible in the future by going for future indemnifications to others that just might get it from Crowdin.

Wonder what kind of information that is, and where it goes?

>4.5 **Change of Ownership**

>**Information about Users** and Visitors, including Personal Data, may be **disclosed and** otherwise **transferred to** an **acquirer, successor or assignee as part of any merger, acquisition, debt financing, sale of assets, or similar transaction**, **as well as** in the event of an **insolvency, bankruptcy**, or **receive**rship in which **information** is **transfer**red **to** one or more **third parties** as **one of our business assets** and only **if the recipient of the User** or Visitor Data commits to **a Privacy Policy** that has **terms substantially consistent with this Privacy Policy**.

Even more-so than the rights one can sign away permanently, there can be a contract around it that holds.

Crowdin thinks contributions are a "business asset" that they can use however they like damn well like, even to the point where you can't use it in any other way either as part of the contract.

We could go on. It really is a mess.
There is also an enormous amounts of extra terms and conditions that apply, because it establishes that in its description of connection to others.

So you have to compare the business aspirations between those described in opposing terms and conditions here, to what could converge at the other end. What does this "interest" at the sole whim of who Crowdin collaborates with as its "partners"
Read that with the business interests of both companies for what they call what is now their "asset".
It is sick.

Collecting data in groups instead of individual-targeted info is considered enough to call even collecting data for
targeted ads de-anonymizing. Then they supposedly aren't targeted, and it isn't "spying".
I don't want to allow for my data to be used for that in any scope.
It is perfectly indistinguishable at my end.
Leaving smaller and smaller room for error as datasets converge gives me less privacy.
You specifically allow it, because you need it, and you use it, that is what Crowdin is.

Maybe this is a bit hard to read even with annotations, but if you think about writing fair terms and conditions
where your voluntary labour makes up all of the bulk of what is only "a" the "service".
Whare in your document of fair terms and conditions would "customized advertisements" come up?
So where did they put it?

It is almost easier to read if you assume the worst and read backwards from there.
It all reads like things you could say in a courtroom to legally defend "how are users supposed to know you mean this?"

- Well, it says right there in "Change of Ownership", and that is, "p", oh, what is "p", well "p" is this other thing over here,
- and that is tied to this, but there is no question this is a change of owernship, we believe it to be in conjuction with the terms described. That would be a winning argument AFAIK, and I am not a lawyer nor do I give legal advice.

Fits right in line with other things this company has been caught doing, like pretending to be engineers when writing reviews and actually being PR staff, or talking as users not disclosing their active intentions over and over again in projects.

Full disclosure I on the other hand has not gotten anything other than two t-shirts from Weblate, which works out to be a modestly mediocre salary when you factor in the hours.

And I can already here, no that is with the service only for a very limited scope, etc.

>The **business partners and advertising networks** that serve interest-based advertisements on the Services have limited **access to** a small amount of **information about your profile** and your device, which is necessary to serve you advertisements that are tailored to your apparent interests. It is possible that **they** may **reuse this** small amount of **information on other** sites or **services**.

It is itself selling tracking in the interest of continued advertisement.

These provisions are made specific and so that it is possible to describe a legitimate interest as nothing different from what it also is.

Take a look at https://support.crowdin.com/cookies/ and read the ones for  "Description is not currently available."

VISITOR_PRIVACY_METADATA  Description is currently not available.

helps in delivering a better user experience for the visitors

>_gat_gtag_UA_*	1 minute	Google Analytics sets this cookie to store a unique user ID.

This is what I was talking about earlier.

It literally says "s_partner", "m_partner" and "c_partner", as "description not currently available".
You want to put faith in a closed source SAAS-company that pulls that "trick"?

>_fbp	**3 months**	Facebook sets this cookie to **display advertisements** when either on Facebook or **on a digital platform powered** by Facebook advertising **after visiting** the website.

This is not a translation service, this is a data acquisition and targeting business disservice.

As I said developers are quite familiar with this not because they neccisarily know anything about legal matters, but they can understand bugs in code are often stuff that does what it is supposed to do, and then a little bit more.
Then you find something else over here that does a little bit, and all of a sudden it is a bug too.
You can spot someone making an active effort to just be perfectly sloppy enough to try to get away with it.

There is no single rational reason what details about what you do on Crowdin should be in the hands of the interests behind the thereby attemptedly targeted adds you see as presented by Facebook in the aim to make money on advertisment at both ends.
That is a "transfer of ownership", and you could really call that a "business asset", because it is, and it is everything you do, touch or touch since.

If you get me even started on functionality we will grow old together.

**Reply by Navid200** [reply]
You can create an email address with Google and use instead of your favorite email address if you are worried about your email address getting in the wrong hands.

I chose a user name and password and that's all I had to enter to sign up..  I never had to enter my name or address.

So, I still don't see what the problem is with Crowdin.

**Reply by comradekingu** [reply]
OK, in a perfectly "free" world, I guess those practices are above board.
Can we at least be honest It is a far cry from wanting to inform users about it happening to hide it in there.
To do it all I mean, trying to target advertisement, and selling data to other services that do?
Are we because of that going to say it doesn't matter if one service and one does not? 
What could be worse?

Lets just keep on using the one that does it, no worries?
That is a horrific take on the sanctity of user privacy.
Yes it is full freedom when you can be dumb with it, but we shouldn't allow obvious traps, that btw. are anything but avoided by (of all things) signing up with a Google address.

I mean you just get it from all angles at that point…!?
It is worse.
The idea that using spyware is in any way anonymous (where it should be) because there is more of the same?
This happens when users who don't understand it, and reasonably have no reason to expect being treated that way for doing a service to what are clearer users of the site, in the background.

Almost glanced over that it isn't just your e-mail address this happens to.
It is other data too, and any net that catches you is effective.
Is it OK if it isn't strictly illegal _everywhere_ yet?

It is not OK. It is not comparable, it is a whole different product.
My point stands.

**Reply by Navid200** [reply]
Would you inform the translators that their name and email address may be recorded by Crowdin?
Isn't that common sense about any entity you interact with online anyway?

**Reply by comradekingu** [reply]
Yes. Why do you ask?
That isn't what that did that I had a problem with even.
It is like asking, wouldn't you put the numbers in point 5.4 after point 5.3 if it is the one section betlow it?
Look what it says.

Spying on users is the problem. Targeting users with ads is the problem. It is not limited to inside Crowdin.
Neither are being able to sign up with a throwaway account and the behavior of stealing data is explained by each-other.
It is a strange place to have zero consideration for what one exposes users to in a GPLv3 project.

Crowdin wants rights to translations and target you with ads.
It wants data while that is going on.
It uses that data for its own benefit and monetize it with others.
And what does it do on the landing page of the website, well it about all the things it can do with those users' data and AI.
Not a single word that this is going on.
You don't get to be Weblate if you are not the Weblate in the Weblate terms. 

It would be illegal with the Weblate terms, and that is a better way to do terms and services.

**Reply by Navid200** [reply]
OK,
I think you have made your point.
I assure you the project lead will see your points.
